database/postgresql/schema.sql

```diff
@@ -0,0 +1,53 @@
+-- If grubhub schema does not exist, create it
+CREATE SCHEMA IF NOT EXISTS grubhub AUTHORIZATION phdickson;
```

Comment:
Again, don't think it's a good idea to make the `AUTHORIZATION` info public. A FILL_ME_IN can hide this.